<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartPropertiesDialogForm</class>
 <widget class="QDialog" name="ChartPropertiesDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>506</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chart Style</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="m_componentsListWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="200">
               <red>184</red>
               <green>184</green>
               <blue>184</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="200">
               <red>184</red>
               <green>184</green>
               <blue>184</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="AlternateBase">
             <brush brushstyle="SolidPattern">
              <color alpha="200">
               <red>184</red>
               <green>184</green>
               <blue>184</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="m_tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_applyPushButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_exitButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_componentsListWidget</tabstop>
  <tabstop>m_tabWidget</tabstop>
  <tabstop>m_applyPushButton</tabstop>
  <tabstop>m_exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_exitButton</sender>
   <signal>clicked()</signal>
   <receiver>ChartPropertiesDialogForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
